restapi/src/main/java/io/stargate/web/RestApiActivator.java

```diff
@@ -36,6 +37,8 @@
           "AuthIdentifier",
           System.getProperty("stargate.auth_id", "AuthTableBasedService"));
   private final ServicePointer<Metrics> metrics = ServicePointer.create(Metrics.class);
+  private final ServicePointer<HttpMetricsTagProvider> httpTagProvider =
+      ServicePointer.create(HttpMetricsTagProvider.class);
```

Comment:
nit: This seems to be a common dependency. Would it make sense to introduce a base class for web service and hadle it there?

Reply:
Could be done in https://github.com/stargate/stargate/issues/239?

Reply:
sure